azext_edge/edge/providers/edge_api/dataflow.py

```diff
@@ -16,7 +16,7 @@ class DataflowResourceKinds(ListableEnum):
 
 DATAFLOW_API_V1B1 = EdgeResourceApi(
```

Comment:
We should change the variable names too when it is no longer beta.

Reply:
Oh yeah, will do